ruby/true_automation/lib/true_automation/driver/appium_lib.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../client'
+
+module TrueAutomation
+  module Driver
+    class AppiumLib < Appium::Core::Driver
+      def initialize(opts = {})
+        @delegate_target = self # for testing purpose
+        @automation_name = nil # initialise before 'set_automation_name'
+
+        opts = Appium.symbolize_keys opts
+        validate_keys(opts)
+
+        @custom_url = opts.delete :url
+        @caps = get_caps(opts)
+
+        set_appium_lib_specific_values(get_appium_lib_opts(opts))
+        set_app_path
+        set_appium_device
+        set_automation_name
+
+        extend_for(device: @device, automation_name: @automation_name)
+
+        @ta_client = TrueAutomation::Client.new
+        @remote = ''
+
+        self # rubocop:disable Lint/Void
+      end
+
+      def start_driver(server_url: nil,
```

Comment:
What do you think about just using `args` as params here?

Reply:
How it connect to super if I change it?

Reply:
Also what to do with default values if it will be `args`